lib/puppet/indirector/request.rb

```diff
@@ -9,6 +9,7 @@
 # analogous to an HTTP Request object, except tuned for our Indirector.
```

Comment:
In `lib/puppet/util/connection.rb` in @Sharpie 's commit, he added `require 'puppet/util/warnings'`. Do we need that here too?

Reply:
Charlie's debug_once commit: https://github.com/puppetlabs/puppet/commit/179f3e8e5d0cc059ccc4e77a96e1d828b716015d